README.md

```diff
@@ -54,4 +55,4 @@ node .github/workflows/sync-docs.js terraform-modules "Terraform Modules" module
 node .github/workflows/sync-docs.js azure-devops-templates "Azure DevOps Templates"
 ```
 
-Do not commit the folders that are synced as of now.
\ No newline at end of file
+Do not commit the folders that are synced as of now.
```

Comment:
We don't sync any folders any more. This section can be removed.
It should have been done by the person that disabled the syncing in the first place

Reply:
Removed.